lib/gceimgutils/gceutils.py

```diff
@@ -81,35 +82,50 @@ def find_images_by_name_regex_match(images, image_name_regex, log_callback):
 
 
 # ----------------------------------------------------------------------------
-def get_credentials(project_name=None, credentials_file=None):
+def load_credentials_from_file(credentials_file):
+    with open(credentials_file) as creds_file:
+        credentials_info = json.load(creds_file)
+
+    return credentials_info
+
+
+# ----------------------------------------------------------------------------
+def get_credentials(
+    project_name=None,
+    credentials_file=None,
+    credentials_info=None
+):
     """Get the service account credentials for the given project"""
 
-    if not project_name and not credentials_file:
+    if not project_name and not credentials_file and not credentials_info:
         raise GCEProjectCredentialsException(
-            'Either project name or credentials file path must be given'
+            'Either project name, credentials file path or credentials '
+            'object must be given'
```

Comment:
This message makes no sense if the tool is invoked from the command line, how would one specify the credentials object. I would suggest to separate the handling of the command line from potential use as a library.

Reply:
OK